tic-tac-toe-app/styles.css

```diff
@@ -0,0 +1,32 @@
+html {
+    font-family: monospace;
+}
+
+.container {
+    display: flex;
+    justify-content: center;
+}
+
+.gameCell {
```

Comment:
There is also a `gameCell` style 👇 . Which set of styles to you prefer to use for the "gameCell" class?